packages/forms-web-app/src/controllers/selected-appeal/final-comments-details/format-final-comment-details.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * @typedef {import('appeals-service-api').Api.AppealFinalComment} AppealFinalComment
+ */
+
+const { formatDocumentDetails } = require('@pins/common');
+
+/**
+ * @param {AppealFinalComment[]} comments
+ */
+exports.formatFinalComment = (comments) => {
+	return comments.map((comment, index) => {
+		const documents = comment.FinalCommentDocuments?.map((doc) => doc.Document);
+		const documentDetails = formatDocumentDetails(documents, 'appealFinalComment'); //Update when data model set
+
+		const fullText = comment.comments;
+		const truncated = fullText.length > 150;
+		const truncatedText = truncated ? fullText.substring(0, 150) + '...' : fullText;
```

Comment:
do I remember this being done somewhere else, either way seems a good candidate for a utility function